### PR TITLE
Add friends list

### PR DESCRIPTION
**Pull request by altheaden**
Added a friends list to each profile, so a user's friends will be listed with links to their respective profiles. 

I also added a change that will eventually check if a user is logged in to see if the header should display the user's name and pfp or the login button. There is currently only a dev workaround to manually switch from one to the other, until I get logging in sorted on the template side.

### CODE REVIEW

File: src/templates/profile.html
Comment (franklindyer) [inline src/templates/profile.html:1]: Nitpick, but is there a reason you are using `h3` and `h4` for the username title and friends list title? In the spirit of making html semantic and css cosmetic, if you're just doing this for their appearances, you might instead change their appearances by adding styles for their classes. Maybe a little weird to have a `h3` and `h4` when the page doesn't have any `h1`s or `h2`s. (Although I sloppily do the same thing on some of my blog posts...)

**Review by franklindyer — COMMENTED** [review]
The indentation is kinda irregular in this file, I realize it's probably going to be overhauled eventually but can you do something about
- the one extremely long line with the `img` tag
- the `a` and `input` that have an extra indent at the end (unless there's a reason)

File: src/templates/profile.html
Comment (altheaden) [reply src/templates/profile.html:1]: Good idea.

Comment (altheaden) [reply src/templates/profile.html:1]: Also, please do nitpick, I really appreciate having good formatting in code :)

**Comment by altheaden** [comment]
@franklindyer I made those changes, can you give it another look and merge if you like it?

**Pull request merged**